zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -32,8 +34,9 @@ pub fn init_wallet_db<P>(wdb: &WalletDb<P>) -> Result<(), rusqlite::Error> {
     wdb.conn.execute(
         "CREATE TABLE IF NOT EXISTS accounts (
             account INTEGER PRIMARY KEY,
-            extfvk TEXT NOT NULL,
-            address TEXT NOT NULL
+            extfvk TEXT,
+            address TEXT,
+            transparent_address TEXT
         )",
```

Comment:
Should this be a prepared and cached statement? (Ditto for other SQL statements in this file.)